cluster/log-dump.sh

```diff
@@ -132,7 +132,7 @@ function save-logs() {
     if log-dump-ssh "${node_name}" "sudo systemctl status kubelet.service" &> /dev/null; then
         log-dump-ssh "${node_name}" "sudo journalctl --output=cat -u kubelet.service" > "${dir}/kubelet.log" || true
         log-dump-ssh "${node_name}" "sudo journalctl --output=cat -u docker.service" > "${dir}/docker.log" || true
-        log-dump-ssh "${node_name}" "sudo journalctl --output=cat -k" > "${dir}/kern.log" || true
+        log-dump-ssh "${node_name}" "sudo journalctl --output=short-precise -k" > "${dir}/kern.log" || true
```

Comment:
The default is actually `--output=short`. `short-precise` has microsecond precision (i.e., `17:13:44` vs `17:13:44.338525`)